CONTRIBUTING.md

```diff
@@ -99,3 +105,52 @@ instructions for installing it.
 
 When opening a new issue, try to roughly follow the commit message format
 conventions above.
+
+## Developing
```

Comment:
Great section! Since we talked about OS env vars, how would a dev enable the pretty printer easily? Would be nice to explain this here too.

Reply:
Good idea. CONTRIBUTING now contains a mini developer's user guide.